### PR TITLE
[MODORDERS-820]-Added intigration tests.

### PR DESCRIPTION
**Pull request by SinghAdes**
## Purpose
_Describe the purpose of the pull request._
https://issues.folio.org/browse/MODORDERS-820
## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

**Comment by SinghAdes** [comment]
image

**Comment by SinghAdes** [comment]
image

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/item-and-holding-operations-for-order-flows.feature [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    * set poLine.fundDistribution[0].fundId = fundId
+    * remove poLine.locations[0].locationId
+    * set poLine.locations[0].holdingId = initialHoldingId
+    Given path 'orders/order-lines'
+    * configure headers = headersUser
+    And request poLine
+    When method POST
+    Then status 201
+    * def createdLine = $
+    * def poLineNumber = createdLine.createdLine
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/item-and-holding-operations-for-order-flows.feature:61]: Seems this line should be corrected, it is not po line number, also it is not used, so we can delete

**Review by SerhiiNosko — APPROVED** [review]

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/item-and-holding-operations-for-order-flows.feature (hunk 1 above)
Comment (SinghAdes) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/item-and-holding-operations-for-order-flows.feature]: deleted

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**